cmd/ocm/gcp/create-wif-config.go

```diff
@@ -222,11 +233,30 @@ func createWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 		log.Printf("Failed to create IAM service accounts: %s", err)
 		return fmt.Errorf("To clean up, run the following command: ocm gcp delete wif-config %s", wifConfig.ID())
 	}
+
+	//The IAM API is eventually consistent. If the user created the service
+	//accounts needed for cluster deployment within too brief a period, then
+	//our backend will not yet have access to it. To avoid confusing error
+	//messages being returned, we will verify that the backend can see the
+	//resources before we consider the wif-config creation complete.
+	if err := utils.RetryWithBackoffandTimeout(func() (bool, error) {
+		log.Printf("Verifying wif-config '%s'...", wifConfig.ID())
+		if err := verifyWifConfig(connection, wifConfig.ID()); err != nil {
+			return true, err
+		}
+		return false, nil
+	}, VerificationTimeoutSeconds, log); err != nil {
+		log.Printf("Failed to verify wif-config resources: %s", err)
+		return fmt.Errorf("To clean up, run the following command: ocm gcp delete wif-config %s", wifConfig.ID())
```

Comment:
does it have to be cleaned up ? they could retry by running verify too, isnt ?

Reply:
As discussed during OH , please reconsider the msg reported to the user based on whether its a creation error or verification error

Reply:
as discussed during office hours, I changed this error message to suggest that the user update if verification timed out.